source/refunding_payments/index.html.md.erb

```diff
@@ -164,12 +163,11 @@ refund attempt that fails like this with an error code is not assigned a
 href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/payment-id/find-payment-refund-by-id"
 target="blank">Find payment refund by ID</a> (link opens in new window)).
 
-If accepted by GOV.UK Pay, a refund may still go on to fail at the PSP. This
-may happen if the card involved is cancelled or has expired, or if your
-account with the PSP does not have enough funds to cover the refund.
+If a refund is accepted by GOV.UK Pay, it may still go on to fail at the PSP.
+This may happen if the card involved is cancelled or has expired, or if your
```

Comment:
In the 'refund destination' section - "If the original card expired or was canceled, the bank will attempt to credit the customer’s new card with the refund" - perhaps we could clarify this to make it clear that a cancelled or expired card isn't the sole reason why the refund attempt failed? Is there room for confusion here?